pages/common/mysqldump.md

```diff
@@ -10,3 +10,11 @@
 - Restore a backup, user will be prompted for a password:
 
 `mysql -u {{user}} --password -e "source {{filename.sql}}" {{database_name}}`
+
+- Backup all databases and redirect it to a file, user will be prompted for a password:
+
+`mysqldump -u {{user}} -p --all-databases > {{filename.sql.gz}}`
```

Comment:
Remove .gz

Reply:
Fixed

Reply:
I still don't see it in the latest commit - https://github.com/tldr-pages/tldr/pull/3177/files#diff-d2e51c131025b22a22fe2f70f93895a3R16

Reply:
Sorry, nice catch! It's here https://github.com/pengux/tldr/commit/6e17cbf5e286bba86d3504b97f1c6a2e410901fb